IIS-Sample.yaml

```diff
@@ -28,4 +28,18 @@ spec:
           - containerPort: 80
   selector:
     matchLabels:
-      app: iis-sample
\ No newline at end of file
+      app: iis-sample
+
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: windows-service
+spec:
+  selector:
+    app: windowsservice
```

Comment:
I believe app here should be `iis-sample` and we could rename the service to `iis-sample-service`

Reply:
Now that I think about it does iis-sample suffice or do you reckon we could change it to something more meaningful?

Also, any comments on deploying an IIS image  bundle with LogMonitor and show the logs? overkill?

Reply:
Bundling Log Monitor for the sample makes sense